README.md

```diff
@@ -1 +1,109 @@
-# persistentvolumeclaim-storageclass-policy
+# Restrict StorageClasses in PersistentVolumeClaims
+
+This Kubewarden policy is designed to enhance the security and manageability of
+Kubernetes clusters by preventing the use of certain storage classes within
+`PersistentVolumeClaim` (PVC) objects. The policy provides an option to
+configure a fallback storage class, offering a seamless alternative when a
+denied storage class is requested.
+
+## Policy Objective
+
+The main goal of this policy is to restrict the usage of specific storage
+classes in PersistentVolumeClaims. Optionally, the policy can mutate the PVC to
+utilize a designated fallback storage class if a restricted storage class is
+requested. This policy does not evaluate PVCs that do not specify a
+`storageClassName`, as Kubernetes defaults to the cluster's default storage
+class in such cases.
```

Comment:
I think we can get rid of this section, it's a repetition of the previous section. The only missing line (from the previous section) is the note about the default storage class